src/elements/content-sidebar/activity-feed/common/activity-message/__tests__/ActivityMessage.test.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { mount, shallow } from 'enzyme';
 
-import ActivityMessage from '../ActivityMessage';
+import { ActivityMessage } from '../ActivityMessage';
```

Comment:
Should we be testing the default export by wrapping it in a FeaturesProvider, instead? It's what other components will use.